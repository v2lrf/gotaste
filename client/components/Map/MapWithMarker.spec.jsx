import React from 'react'
import { MapWithMarker } from '.'

describe('MapWithMarker', () => {
  let component
  let props
  beforeEach(() => {
    props = {
      latitude: 55.6753,
      longitude: 12.5703
    }

    component = render(<MapWithMarker {...props} />)
  })

  describe('Snapshots', () => {
    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
