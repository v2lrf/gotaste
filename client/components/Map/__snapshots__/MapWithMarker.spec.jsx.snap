// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapWithMarker Snapshots should match the snapshot 1`] = `
<div
  style="position:relative;width:100%;height:200px"
>
  <div
    class=""
    style="position:absolute;width:100%;height:100%;overflow:hidden;visibility:inherit"
  />
  <div
    style="overflow:visible;width:0"
  />
</div>
`;
